content/chapters/compute/lecture/quiz/threads-shared-data.md

```diff
@@ -0,0 +1,16 @@
+## Question Text
+
+Where cannot we place data shared by threads?
```

Comment:
I think the statement could be improved, it's pretty weird. Maybe use just "Which section stores per-thread variables, i.e. non-shared variables?"